HW_3/task_1.py

```diff
@@ -0,0 +1,12 @@
+# Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+
+lst_items = [11, 'два', 55, 'один', 11, 'три', 33, 'два', 55, 'один', 77, 77, 77, 'ТРИ']
+lst_duplicate = []
+VALUE_DUPLICATE = 2
+
+for i in lst_items:
+    if lst_items.count(i) == VALUE_DUPLICATE:
+        lst_duplicate.append(i)
+print(list(set(lst_duplicate)))
```

Comment:
Довольно много итераций, высокая сложность, чтобы немного уменьшить сложность можно сделать так.
```suggestion
for i in set(lst_items):
    if lst_items.count(i) == VALUE_DUPLICATE:
        lst_duplicate.append(i)
print(lst_duplicate)
```